addons/account_edi_ubl_cii/models/account_edi_xml_ubl_bis3.py

```diff
@@ -124,7 +124,7 @@ def _get_partner_party_identification_vals_list(self, partner):
         # EXTENDS account.edi.xml.ubl_21
         vals = super()._get_partner_party_identification_vals_list(partner)
 
-        if partner.country_code == 'NL':
+        if partner.country_code == 'NL' and 'l10n_nl_oin' in partner._field:
```

Comment:
@JulienVR Please check
```suggestion
        if partner.country_code == 'NL' and 'l10n_nl_oin' in partner._fields:
```